extensions/interactions/base.py

```diff
@@ -89,6 +89,11 @@ class BaseInteraction(object):
     # containing this interaction. These should correspond to names of files in
     # feconf.DEPENDENCIES_TEMPLATES_DIR. Overridden in subclasses.
     _dependency_ids = []
+    # Additional JS library dependencies that should be loaded in pages
```

Comment:
Ah, I see. For now, I think it's fine to combine them and store them in the same folder. I know we can do things at a finer-grained level, but let's wait to see whether it actually is a problem in practice before adding that complexity. We should be caching most of these files anyway.